pages/conference/keynotes.vue

```diff
@@ -62,12 +62,12 @@
                 </h2>
                 <tabs class="keynote__tabs">
                     <tab :title="$t('terms.bio')">
-                        <div class="pb-4 whitespace-pre-line">
```

Comment:
why does this class need to be removed? i thought arranging the content by paragraph is desired.

before the change
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/24987826/183970418-e7aa49ea-046a-482d-b436-55981ae94da5.png">


after the change
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/24987826/183970599-21d869dd-8d4d-42b6-b9c5-d5d8e79f85ea.png">

Reply:
Ahh, sorry. I didn't explain it and also had mistakes.
Because the tab use 24px for padding in figma, and I think it's very hard to fit the spec by pre-line. 
So the padding maybe a way to go.
And I actually don't remember why I replaced p-4 with py-4 in core/tabs/Tabs XD [66b717c](https://github.com/pycontw/pycontw-frontend/pull/312/commits/66b717c205136ae677be2a72a7129521ad315d03). 
Fixed it by [031d302](https://github.com/pycontw/pycontw-frontend/pull/312/commits/031d302489fed9d336929905d030cde075fe2d99).

Reply:
[8baed6d](https://github.com/pycontw/pycontw-frontend/pull/312/commits/8baed6dc15b571de9885b2593deb30a8cec8bd57)